### PR TITLE
Update dependencies & travis

### PR DESCRIPTION
**Pull request by hildjj**
urllib updated to use any-promise, fixing the install warning.

### CODE REVIEW

**Comment by hildjj** [comment]
Tests pass fine here.  There's likely an issue with output HTTP requests from Travis.

File: .travis.yml [hunk 1]
```diff
@@ -1,5 +1,7 @@
 language: node_js
+sudo: false
 node_js:
   - "0.11"
+  - "node"
```
Comment (dead-horse) [inline .travis.yml]: better change to `4`, `5`, `6`, others LGTM

**Comment by dead-horse** [comment]
1.1.3

**Comment by hildjj** [comment]
Why not point the test URLs at gh-pages?

**Comment by dead-horse** [comment]
@hildjj because the test case will timeout in ___china___ :(

**Pull request merged**